solidity-v1/dashboard/src/pages/threshold/HowItWorksPage.jsx

```diff
@@ -0,0 +1,136 @@
+            </List.Item>
+            <List.Item className="mb-1">
+              Migrate your liquid KEEP to T in the portal.
+            </List.Item>
+            <List.Item className="mb-1">
+              You can upgrade whatever amount of tokens you want in one single
+              transaction.
+            </List.Item>
+          </List.Content>
+        </List>
+
+        <List className="mt-3">
+          <List.Title className="h5 text-violet-80">
+            Staked Granted Tokens
+          </List.Title>
+          <List.Content className="bullets bullets--violet-80 text-grey-60">
+            <List.Item className="mb-1">
+              If you are staking a part of your granted tokens you can either
+              opt to undelegate your tokens or continue staking on Keep Network.
+            </List.Item>
+            <List.Item className="mb-1">
+              If you undelegate, there is a 60 day undelegation period.
+            </List.Item>
+            <List.Item className="mb-1">
+              After 60 days, you can withdraw your tokens and upgrade your KEEP
+              to T if they are available to release from your token grant.
+            </List.Item>
+          </List.Content>
+        </List>
+      </section>
+
+      <section className="tile threshold-how-it-works-page__upgrade-liquid-keep">
+        <h3 className="mb-1">Upgrade Liquid KEEP Tokens</h3>
+        <p className="text-grey-60">
+          You can upgrade any amount of your liquid KEEP tokens in one
+          transaction on the Threshold dapp.
+        </p>
+        <div className="threshold-how-it-works-page__upgrade-liquid-keep__button-container">
+          <Icons.TTokenSymbol />
+          <h4 className="button-container__title">Threshold</h4>
+          <a
+            href={LINK.tbtcDapp}
+            rel="noopener noreferrer"
+            target="_blank"
+            className="btn btn-secondary btn-md"
+            style={{ marginLeft: "auto" }}
+          >
+            go to dapp ↗
+          </a>
+        </div>
+      </section>
+
+      <section className="tile threshold-how-it-works-page__upgrade-staked-keep">
+        <h3 className="mb-1">Upgrade Staked KEEP Tokens</h3>
+        <List>
+          <List.Title className="h5 text-violet-80">Staked Tokens</List.Title>
+          <List.Content className="bullets bullets--violet-80 text-grey-60">
+            <List.Item className="mb-1">
+              You can either undelegate your tokens or continue staking on Keep
+              Network.
+            </List.Item>
+            <List.Item className="mb-1">
+              If you undelegate, there is a 60 day undelegation period until you
+              can withdraw your tokens.
+            </List.Item>
+            <List.Item className="mb-1">
+              After 60 days, you can withdraw your tokens and upgrade your KEEP
+              to T.
+            </List.Item>
+          </List.Content>
+        </List>
```

Comment:
Let's add some margin-bottom.

Reply:
Fixed in 8617f786cc6447eb39e4bbcd2d69bb253716bf62